web_widget_radio_tree/README.rst

```diff
@@ -0,0 +1,70 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=====================
+Web widget radio tree
+=====================
+
+This module allows to use input radio in a tree view inside a form, in order to ensure the user marks only one record.
```

Comment:
judging from the code, you also provide a form widget, so this seems an unnecessary restriction to me